capture_service/android_application.cc

```diff
@@ -454,13 +461,20 @@ VulkanCliApplication::~VulkanCliApplication()
 absl::Status VulkanCliApplication::Setup()
 {
     RETURN_IF_ERROR(GrantAllFilesAccess());
-    RETURN_IF_ERROR(m_dev.Adb().Run(absl::StrFormat("shell mkdir -p %s", kVulkanGlobalPath)));
-    RETURN_IF_ERROR(
-    m_dev.Adb().Run(absl::StrFormat(R"(push "%s" "%s")",
-                                    ResolveAndroidLibPath(kVkLayerLibName, "").generic_string(),
-                                    kVulkanGlobalPath)));
-    RETURN_IF_ERROR(
-    m_dev.Adb().Run(absl::StrFormat("shell setprop debug.vulkan.layers %s", kVkLayerName)));
+    if (m_gfxr_enabled)
+    {
+        RETURN_IF_ERROR(GfxrSetup());
+    }
+    else
+    {
+        RETURN_IF_ERROR(m_dev.Adb().Run(absl::StrFormat("shell mkdir -p %s", kVulkanGlobalPath)));
```

Comment:
maybe better to have a func to be consistent with `GfxrSetup`, maybe something like `CLISetup`?